src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -349,6 +349,31 @@ describe('StoryPromo Container', () => {
       );
     });
 
+    [
+      {
+        service: 'pashto',
+        expectationFirstJan2020: '۱۱ مرغومی ۱۳۹۸ - جنوري ۱، ۲۰۲۰',
+      },
+      {
+        service: 'persian',
+        expectationFirstJan2020: '۱۱ دی ۱۳۹۸ - ۱ ژانویه ۲۰۲۰',
+      },
+    ].forEach(({ service, expectationFirstJan2020 }) => {
+      it(`should render time element with multiple calendars for ${service} Story Promo`, () => {
+        const firstJan2020 = 1577836800000;
+        const newItem = {
+          ...cpsItem,
+          timestamp: firstJan2020,
+        };
+        const persianContainer = render(
+          <WrappedStoryPromo service={service} item={newItem} />,
+        ).container;
```

Comment:
```suggestion
        );
```
Following Denis' suggestion

Reply:
Thanks, this change was made.